lib/Gocdb_Services/Config.php

```diff
@@ -567,4 +567,59 @@ public function getEmailTo()
 
         return $emailTo;
     }
+
+    public function getIdentityProvidersInfo(): array
+    {
+        $localInfo = $this->GetLocalInfoXML();
+        $identityProviders = [];
+
+        if (!empty($localInfo->identity_providers->provider)) {
```

Comment:
What happens here if `identity_providers` is omitted in it's entirety from the `local_config.xml`?

Reply:
identity_providers -> this is anyway optional. But if passed thru xml then it MUST have provider(This being idp, name, and other attributes) info.

This is just a safe check. Just in case if Null case is referring to any method(s) due to which it may cause errors.